opencti-platform/opencti-front/src/private/components/settings/workflow/StatusCreation.js

```diff
@@ -87,152 +84,123 @@ const statusCreationMutation = graphql`
-            <Form>
-              <Dialog
-                open={this.state.open}
-                PaperProps={{ elevation: 1 }}
-                onClose={this.handleClose.bind(this)}
-                fullWidth={true}
-              >
-                <DialogTitle>{t('Create a status')}</DialogTitle>
-                <DialogContent>
-                  <QueryRenderer
-                    query={statusCreationStatusTemplatesQuery}
-                    render={({ props }) => {
-                      if (props && props.statusTemplates) {
-                        const statusTemplatesEdges = props.statusTemplates.edges;
-                        const statusTemplates = R.map(
-                          (n) => n.node,
-                          statusTemplatesEdges,
-                        );
-                        return (
-                          <Field
-                            component={SelectField}
-                            variant="standard"
-                            name="template_id"
-                            label={t('Name')}
-                            fullWidth={true}
-                            containerstyle={{ width: '100%' }}
-                          >
-                            {statusTemplates.map((statusTemplate) => (
-                              <MenuItem
-                                key={statusTemplate.id}
-                                value={statusTemplate.id}
-                              >
-                                {t(`status_${statusTemplate.name}`)}
-                              </MenuItem>
-                            ))}
-                          </Field>
-                        );
-                      }
-                      return <div />;
-                    }}
-                  />
-                  <Field
-                    component={TextField}
-                    variant="standard"
-                    name="order"
-                    label={t('Order')}
-                    fullWidth={true}
-                    type="number"
-                    style={{ marginTop: 20 }}
-                  />
-                </DialogContent>
-                <DialogActions>
-                  <Button onClick={handleReset} disabled={isSubmitting}>
-                    {t('Cancel')}
-                  </Button>
-                  <Button
-                    color="secondary"
-                    onClick={submitForm}
-                    disabled={isSubmitting}
-                  >
-                    {t('Create')}
-                  </Button>
-                </DialogActions>
-              </Dialog>
-            </Form>
-          )}
-        </Formik>
-      </div>
-    );
-  }
-}
+  const onReset = () => {
+    handleClose();
+  };
 
-StatusCreation.propTypes = {
-  display: PropTypes.bool,
-  classes: PropTypes.object,
-  theme: PropTypes.object,
-  t: PropTypes.func,
+  return (
+    <div style={{ display: display ? 'block' : 'none' }}>
```

Comment:
@Kedae It's not better to use the condition direclty on the parent/calling component to show or not this component ?

Reply:
Absolutely, I've missed that.